### PR TITLE
add checkout tests

### PR DESCRIPTION
**Pull request by Test01-Parn**

### CODE REVIEW

File: tests/e2e/sauceDemo/003-checkout.spec.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	test('Checkout with many products should successfully create an order', async () => {
+		await inv.addProductToCart('Sauce Labs Backpack');
+		await inv.addProductToCart('Sauce Labs Bike Light');
+		await inv.addProductToCart('Sauce Labs Bolt T-Shirt');
+		await checkout.cartIconButton.click();
+		await checkout.checkoutButton.click();
+		await checkout.fillPersonalInformation();
+		await checkout.continueButton.click();
+		const checkoutProductNames = await checkout.getProductNames();
+		expect(checkoutProductNames[0]).toStrictEqual('Sauce Labs Backpack');
```
Comment (pktdvTA1) [inline tests/e2e/sauceDemo/003-checkout.spec.ts]: should be `.toBe()`

and other 3 assertions below as well

File: tests/e2e/sauceDemo/003-checkout.spec.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		expect(checkoutProductNames[1]).toStrictEqual('Sauce Labs Bike Light');
+		expect(checkoutProductNames[2]).toStrictEqual('Sauce Labs Bolt T-Shirt');
+		await checkout.finishButton.click();
+		await expect(checkout.title).toHaveText('Checkout: Complete!');
```
Comment (pktdvTA1) [inline tests/e2e/sauceDemo/003-checkout.spec.ts]: could this be a dedicated assertion under CheckoutAssertion class ? for reusability.

**Comment by pktdvTA1** [comment]
dont forget husky 🐶(`npm i`)

File: tests/e2e/sauceDemo/003-checkout.spec.ts (hunk 2 above)
Comment (Test01-Parn) [reply tests/e2e/sauceDemo/003-checkout.spec.ts]: only for line 32? or 29, 30, 32

Comment (pktdvTA1) [reply tests/e2e/sauceDemo/003-checkout.spec.ts]: only this one (may have another check like other label as well).

**Pull request merged**